### PR TITLE
Adding API documentation for /databases/db/ca

### PR DESCRIPTION
**Pull request by rpmoore**
Adds the get CA call to get certificates for securing database traffic

### CODE REVIEW

**Comment by ChiefMateStarbuck** [comment]
!deploy

**Review by ChiefMateStarbuck — APPROVED** [review]
clean.

File: specification/resources/databases/examples/curl/get_ca.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+lang: cURL
+source: |-
+  curl -X POST \
```
Comment (andrewsomething) [inline specification/resources/databases/examples/curl/get_ca.yml:3]: @rpmoore  Looks like this should be a GET?

Comment (rpmoore) [reply specification/resources/databases/examples/curl/get_ca.yml:3]: Oh, good catch, I'll fix that.

Comment (rpmoore) [reply specification/resources/databases/examples/curl/get_ca.yml:3]: I'll do it in another PR.

Comment (rpmoore) [reply specification/resources/databases/examples/curl/get_ca.yml:3]: https://github.com/digitalocean/openapi/pull/543

Comment (andrewsomething) [reply specification/resources/databases/examples/curl/get_ca.yml:3]: Thanks!

**Pull request merged**